### PR TITLE
Clarify that enum is const

### PR DESCRIPTION
**Pull request by baruchiro**
## Summary

My friend read this doc and ask me why he can't add values to enum as runtime.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/enum.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.assetid: bbeb9a0f-e9b3-41ab-b0a6-c41b1a08974c
 ---
 # enum (C# Reference)
 
-The `enum` keyword is used to declare an enumeration, a distinct type that consists of a set of named constants called the enumerator list.  
+The `enum` keyword is used to declare a const enumeration, a distinct type that consists of a set of named constants called the enumerator list.
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/enum.md]: I would prefer adding a sentence that clarifies the **enumerator list**.  Consider this:

> Enumerations cannot define runtime behavior. They define a set of compile-time constant values.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for making this change @baruchiro 

We appreciate it. I read the change you proposed, and I have a suggestion that I think better conveys your intent. Do you agree? If so, just make that change on your branch and it will update your PR.

File: docs/csharp/language-reference/keywords/enum.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/keywords/enum.md]: @BillWagner @baruchiro 
I would drop **list** and define **enumerator** instead, as (1) "enumerator" word is used throughout the article further to refer to individual named constant, (2) now we refer to the same thing by two names: set and list (which might add to confusion). So, what about
> The `enum` keyword is used to declare an enumeration, a distinct type that consists of a set of named constants, or **enumerators**.

Furthermore, "a const enumeration" might be confusing. @baruchiro did you mean that that "set of named constants" is fixed at compile time and one cannot add new elements to the set at run time? Then, what about the following update:
> The `enum` keyword is used to declare an enumeration, a distinct type that consists of a predefined set of named constants, or **enumerators**.

Or:
> Use the `enum` keyword to declare an enumeration, a distinct type that consists of a set of named constants, or **enumerators**. The set of enumerators is defined at compile time.

Comment (baruchiro) [reply docs/csharp/language-reference/keywords/enum.md]: I think the problem is with the word **"Enumerator"**.
In C#, the common use is in collections, while the current subject is called **"enum"**.

**Pull request closed without merging**